src/modules/Logs/action.js

```diff
@@ -54,11 +54,7 @@ export const sendLogs = (text: string) => async (dispatch: Dispatch, getState: G
       // Wallet info
       const wallet = currencyWallets[walletId]
       if (wallet && logOutput.loggedInUser) {
-        const currencyCode = wallet.currencyInfo.currencyCode
-          ? wallet.currencyInfo.currencyCode
-          : typeof wallet.currencyCode === 'string'
-          ? wallet.currencyCode
-          : ''
+        const currencyCode = wallet.currencyInfo.currencyCode ?? ''
```

Comment:
so we don't need to set it to `wallet.currencyCode` if no `wallet.currencyInfo.currencyCode`?

Reply:
There is no `EdgeCurrencyWallet.currencyCode`. This was a type error the whole time, but Flow couldn't see the problem because the selectors were messing up its type inference.